packages/form-js-viewer/test/spec/render/components/form-fields/Taglist.spec.js

```diff
@@ -0,0 +1,532 @@
+import {
+  fireEvent,
+  render
+} from '@testing-library/preact/pure';
+
+import Taglist from '../../../../../src/render/components/form-fields/Taglist';
+
+import { createFormContainer } from '../../../../TestHelper';
+
+import { WithFormContext } from './helper';
+
+const spy = sinon.spy;
+
+let container;
+
+
+describe('Taglist', function() {
+
+  beforeEach(function() {
+    container = createFormContainer();
+  });
+
+  afterEach(function() {
+    container.remove();
+  });
+
+
+  it('should render', function() {
```

Comment:
Instead of having one big render test, we could check for individual parts

```js
it('should render');
it('should render tags');
it('should render tag');
....
```

Reply:
As you basically did with other parts (description) 👍